mp/src/game/shared/da/da_player_shared.cpp

```diff
@@ -799,7 +799,7 @@ void CDAPlayerShared::StartSliding(bool bDiveSliding)
 void CDAPlayerShared::EndSlide()
 {
 	// If it was long enough to notice what it was, then train the slide.
-	if (gpGlobals->curtime > m_flSlideTime + 1)
```

Comment:
This is fine, but FYI: the way I usually think of this is, m_flSlideTime + 1 is when the timer expires, so we're waiting for curtime to pass that. The other one is legit pretty confusing though.